team-telnyx/webrtc
# Add React client context provider and hooks

Adds additional TelnyxRTC abstractions via hooks and Context.

## 📝 To Do


## ✋ Manual testing

1. Run `npm start`
2. [Generate a token](https://developers.telnyx.com/docs/v2/webrtc/quickstart) and save as `REACT_APP_TELNYX_LOGIN_TOKEN` in a `.env` file.
3. Open new shell and `cd example`
4. Start example app with `npm start`. Verify localhost:3000 matches screenshot (below)

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

<img width="267" alt="Screen Shot 2020-11-10 at 4 36 16 PM" src="https://user-images.githubusercontent.com/4672952/98750521-0e1e2580-2373-11eb-9786-1c333667d4a6.png">
